src/Controls/src/Core/Compatibility/Handlers/FlyoutPage/iOS/PhoneFlyoutPageRenderer.cs

```diff
@@ -205,18 +205,20 @@ bool shouldReceive(UIGestureRecognizer g, UITouch t)
 		public override void ViewWillTransitionToSize(CoreGraphics.CGSize toSize, IUIViewControllerTransitionCoordinator coordinator)
 		{
 			base.ViewWillTransitionToSize(toSize, coordinator);
-
-			if (FlyoutOverlapsDetailsInPopoverMode)
+			if (!OperatingSystem.IsMacCatalyst())
```

Comment:
The Flyout state will also be maintained when resizing the window for example with this change. What is the behavior on Windows? While the change seems fine to me, I do think the behavior should be aligned between platforms.